src/libgrate/fragment_asm_parser.y

```diff
@@ -765,6 +767,19 @@ TEX_INSTRUCTION:
 	|
 	;
 
+TEX_LOD_REG:
+	',' T_ROW_REGISTER
+	{
+		asm_tex_instructions[asm_fs_instructions_nb].enable_bias = 1;
+		yyval.u = $2;
+	}
+	|
+	{
+		asm_tex_instructions[asm_fs_instructions_nb].enable_bias = 0;
+		yyval.u = 0xDEAD;
```

Comment:
Indeed, differentiating instructions by opcode should be more expressive and there will be more opcodes in the future. We should probably omit the 3-rd cubemap coord source register from the current TEX instruction and use it in the TEX2 instruction opcode (following TGSI ISA) once will add support for cubemaps. 

 I'll re-spin the series.